vimeo-networking/src/main/java/com/vimeo/networking/VimeoService.java

```diff
@@ -73,7 +73,7 @@
     @POST("oauth/authorize/client")
     Call<VimeoAccount> authorizeWithClientCredentialsGrant(@Header("Authorization") String authHeader,
                                                            @Field("grant_type") String grantType,
-                                                           @Field("scope") String scope);
+                                                           @Field("mScope") String scope);
```

Comment:
This is incorrect - we need to send `scope` not `mScope` to the API. The same applies to elsewhere in this file, but I am just going to comment on this one.